analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd

```diff
@@ -41,9 +41,7 @@ if(!dir.exists(output_dir)) {
 ```
 
 ```{r}
-# TODO: the consensus SEG file is not currently in the data download -- when it
-# gets included we will have to change the file path here
-consensus_seg_file <- file.path("..", "copy_number_consensus_call", "results", 
+consensus_seg_file <- file.path("..", "..", "data", 
```

Comment:
@jharenza and @kgaonkar6 - is there any reason (that you are aware of) that prohibits me from making this change at this point?

Reply:
I do not think so - the latest file is in the download.

Reply:
We probably need to wait until all the v20 changes go through though? Or would just getting #1123 through -> updating the release (if not done yet) be sufficient?

Reply:
I've updated v20 release data with the latest consensus seg with #1123 `b9284650be04df3538e6c6dba29b8eb0  pbta-cnv-consensus.seg.gz`

I added the relative path so that we can run the code while running all the preprocessing steps of subtyping, so maybe we can add a logic to change to relative path or not  within the if (params$base_run ==0)

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/a6281c2aebfc69213cf8cc0ff9b2b246b96ea294/analyses/focal-cn-file-preparation/02-add-ploidy-consensus.Rmd#L48-L58

Reply:
In which scenario would you use the file in `analyses/copy_number_consensus_call/results`? When you use `pbta-histologies-base.tsv`?

Reply:
yes, we will rerun the consensus seg file module first so the latest consensus seg file can be used as input for the focal-cn module